osquery/runtime.go

```diff
@@ -630,9 +630,6 @@ func (o *OsqueryInstance) Query(query string) ([]map[string]string, error) {
 
 // kill process group kills a process and all its children.
 func killProcessGroup(cmd *exec.Cmd) error {
-	pgid, err := syscall.Getpgid(cmd.Process.Pid)
-	if err == nil {
-		return syscall.Kill(-pgid, 15)
-	}
-	return errors.Wrap(err, "get PGID")
+	err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL)
```

Comment:
Do we know that the pgid is the same as the process pid?

Reply:
Yes, at least for posix this behavior is true when you apply 
https://github.com/kolide/launcher/blob/8a58d284a1f0152da0523374f53c8d7260323a96/osquery/runtime.go#L448-L449

Reply:
>  The setpgid() function shall either join an existing process group or create a new process group within the session of the calling process. The process group ID of a session leader shall not change. Upon successful completion, the process group ID of the process with a process ID that matches pid shall be set to pgid. As a special case, if pid is 0, the process ID of the calling process shall be used. Also, if pgid is 0, the process ID of the indicated process shall be used.

Reply:
Okay cool thank you. There's already a lot of POSIX dependency in here.